### DISCUSSION TITLE
Video Exercise:Chapter 1: PyTorch Workflow, Problem with nn.Linear

### DISCUSSION THREAD

**Discussion by tonyhaya** [mrdbourke/pytorch-deep-learning#119] [Q&A]
### I've change the optmizer, loss, make new data
[https://colab.research.google.com/drive/14HzW9GKcxuTVtOR6ydMGOZaFFUeypU5Z#scrollTo=mmFZDzP1FalP](url)
